### ISSUE TITLE
flow disappears from hugging face

### ISSUE THREAD

**Issue by uiaiek32** [langflow-ai/langflow#1942]
I create a flow on langflow 1.0 preview and after a day the flow disappears from hugging face and I have to recreate it and re connect the api

**Comment by uiaiek32** [comment]
where do I look this?

**Comment by uiaiek32** [comment]
I have deployed it on Huggingface

**Comment by uiaiek32** [comment]
where do I find this on Hugging face ?

**Comment by nicoloboschi** [comment]
HuggingFace common space is only meant as playground.
You might find useful the new free hosting cloud solution by DataStax. You can sign up here langflow.datastax.com

**Comment by karlgarske** [comment]
Info above is helpful, but I can be a little more specific with respect to persisting data on HuggingFace. Here are the steps I took:

1. The default free tier is unfortunately not persistent. Go to settings and purchase a plan for persistent storage (I used small). 
2. Your persistent data will mount on /data when the space starts, but you still need to configure the langflow db to use the mount. 
3. In settings go to Environment Variables section and add LANGFLOW_DATABASE_URL with value sqlite:////data/langflow.db and LANGFLOW_SAVE_DB_IN_CONFIG_DIR with value true. 
4. Restart your space. Any new flows you create should be persisted after future restarts.
